key/oracle.go

```diff
@@ -16,15 +15,15 @@ func RetrieveAndStoreOraclePrivKey(ctx context.Context, svc service.Service, enc
 		return err
 	}
 
-	if err := sgx.SealToFile(oraclePrivKeyBz, svc.Config().AbsOraclePrivKeyPath()); err != nil {
+	if err := svc.GetSgx().SealToFile(oraclePrivKeyBz, svc.GetConfig().AbsOraclePrivKeyPath()); err != nil {
 		return fmt.Errorf("failed to seal oraclePrivKey to file. %w", err)
 	}
 
 	return nil
 }
 
 func retrieveOraclePrivKey(ctx context.Context, svc service.Service, encryptedOraclePrivKey []byte) ([]byte, error) {
```

Comment:
```suggestion
func decryptOraclePrivKey(ctx context.Context, svc service.Service, encryptedOraclePrivKey []byte) ([]byte, error) {
```
Although this isn't the part of your PR, please rename this function. The `retrieve` has the same meaning as `search`.